go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/databricks/cli
 
-go 1.23
+go 1.23.0
```

Comment:
What's the significance of this?

Or will it be gone after rebase/merge of main?

Reply:
It was updated by tooling. I reverted this back.